group_vars/all.sample

```diff
@@ -186,6 +186,11 @@ dummy:
 # Object backend
 #osd_objectstore: filestore
 
+# xattrs. by default, 'filestore xattr use omap' is set to 'true' if
+# ext4 if 'osd_mkfs_type' is set to 'ext4'; otherwise it isn't set. This
+# can be set to 'true' or 'false' to explicitly override those defaults.
+# filestore_xattr_use_omap: null
```

Comment:
read the first sentence again, I think a word is missing.
moreover shouldn't we set "false" instead of null?